pkg/trace/agent/agent.go

```diff
@@ -26,7 +26,9 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-const processStatsInterval = time.Minute
+// tagContainersTags specifies the name of the tag which holds key/value
+// pairs representing information about the container (Docker, EC2, etc).
+const tagContainersTags = "_dd.tags.container"
```

Comment:
Naming nit: do we consider the keys of the meta map as a tag?

Reply:
We usually refer to them as tags, both to metrics and meta. We might want to change that down the line to differentiate between metrics tags.